### PR TITLE
Update api

### PR DESCRIPTION
**Pull request by ThoJah**

### CODE REVIEW

**Comment by ThoJah** [comment]
This pull request updates the seeLists.ts from main to the new schema.
The other API files got their adjustment in open pull requests that involved other changes on these files.

**Comment by ThoJah** [comment]
Includes the toplevel initialization of prisma

File: src/pages/api/prisma.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { PrismaClient } from "@prisma/client";
```
Comment (NoHara42) [inline src/pages/api/prisma.tsx:1]: change file to .ts as there is no html in this file

File: src/pages/api/deleteList.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { NextApiRequest, NextApiResponse } from "next";
 import { z, ZodError } from "zod";
-import { Items, List, Prisma, PrismaClient } from "@prisma/client";
+import { PrismaClient } from "@prisma/client";
 
 const prisma = new PrismaClient();
```
Comment (NoHara42) [inline src/pages/api/deleteList.ts:5]: this needs to import the exported prisma client in the prisma.ts file, and not instantiate a new prisma client here

**Review by NoHara42 — CHANGES_REQUESTED** [review]
see changes

File: src/pages/api/deleteList.ts (hunk 2 above)
Comment (ThoJah) [reply src/pages/api/deleteList.ts]: applied

File: src/pages/api/prisma.tsx (hunk 1 above)
Comment (ThoJah) [reply src/pages/api/prisma.tsx:1]: applied

**Review by NoHara42 — APPROVED** [review]
LGTM

**Pull request merged**